scanner/src/main/kotlin/experimental/ScanController.kt

```diff
@@ -87,10 +91,14 @@ class ScanController(
     private val scanResults = mutableMapOf<ScannerWrapper, MutableMap<KnownProvenance, MutableList<ScanResult>>>()
 
     fun addProvenanceResolutionIssue(id: Identifier, issue: OrtIssue) {
+        logger.log(issue.severity.toLog4jLevel()) { issue.message }
+
         provenanceResolutionIssues.getOrPut(id) { mutableListOf() } += issue
     }
 
     fun addIssue(id: Identifier, issue: OrtIssue) {
+        logger.log(issue.severity.toLog4jLevel()) { issue.message }
```

Comment:
BTW, are these issues *additionally* logged else where in a single blow? If so, we should probably remove that other logging. Maybe @mnonnenmacher knows?

Reply:
The issues are created [here](https://github.com/oss-review-toolkit/ort/blob/c4a628d37017fb34e0ba7218f52eb52172cb7397/scanner/src/main/kotlin/experimental/ExperimentalScanner.kt#L183-L191) and they arise if PackageProvenanceResolver throws from [here](https://github.com/oss-review-toolkit/ort/blob/c4a628d37017fb34e0ba7218f52eb52172cb7397/scanner/src/main/kotlin/experimental/PackageProvenanceResolver.kt#L223).